channels/af.m3u

```diff
@@ -37,5 +37,3 @@ http://live.stream.cdn.pamirtv.com/ptv/d0dbe915091d400bd8ee7f27f0791303.sdp/inde
 http://51.210.199.50/hls/stream.m3u8
 #EXTINF:-1 tvg-id="SolhTV.af" tvg-name="Solh TV" tvg-country="AF" tvg-language="" tvg-logo="https://i.imgur.com/bAbNyIt.jpg" group-title="",Solh TV
 http://51.210.199.42/hls/stream.m3u8
-#EXTINF:-1 tvg-id="Tuti.af" tvg-name="Tuti" tvg-country="AF" tvg-language="Pushto" tvg-logo="https://i.imgur.com/JerfIBt.jpg" group-title="",Tuti (480p)
-https://rrsatrtmp.tulix.tv/livecdn827/myStream.sdp/playlist.m3u8
```

Comment:
Dead but official link, may be revived

Reply:
Putting it in Not 24/7 for testings, if it's still online after the next test, we'll keep it. We'll need to keep an eye on it.